homeassistant/components/vallox/fan.py

```diff
@@ -154,13 +177,39 @@ async def async_update(self) -> None:
 
         self._available = True
 
-    #
-    # The fan entity model has changed to use percentages and preset_modes
-    # instead of speeds.
-    #
-    # Please review
-    # https://developers.home-assistant.io/docs/core/entity/fan/
-    #
+    async def _async_set_preset_mode_internal(
+        self, preset_mode: str, update_state_proxy: bool
+    ) -> bool:
```

Comment:
```suggestion
    async def _async_set_preset_mode_internal(self, preset_mode: str) -> bool:
```

Reply:
Good one, this way the function does only one thing, that's cleaner. Thanks :+1: